tests/JGLUE_test.py

```diff
@@ -1,6 +1,8 @@
 import datasets as ds
 import pytest
 
+ds.config.HF_DATASETS_TRUST_REMOTE_CODE = True
```

Comment:
I've never heard of this method of specifying. Thank you for adding the setting! It would be great if you could add a brief comment about how this setting works for future reference.

Reply:
This line changes the default value of the Huggingface dataset.

https://github.com/huggingface/datasets/blob/main/src/datasets/config.py

In several versions of datasets, the `trust_remote_code` argument is not implemented in `load_dataset` function, so I adopted this more concise method.

I'm unfamiliar with the Huggingface dataset, so there may be a better way.